### DISCUSSION TITLE
Will I be refunded the cost of SL Premium renewal as I have a Proton Business account?

### DISCUSSION THREAD

**Discussion by TheGuru7** [simple-login/app#1201] [Q&A]
I have a premium SL account and a Proton Business account.  My premium SL subscription was automatically renewed less than a week ago.  It has now been announced that I can link the SL account to my Proton Business one and get the premium SL features for free.  If I now go ahead and link them, will the system recognise that I was only recently charged a SL subscription?  Will there be a refund?

**Comment by nguyenkims** [comment]
@TheGuru7 hi for refund request, please ping our support on https://app.simplelogin.io/dashboard/support. When linking your account, SimpleLogin will see you have 2 active subscriptions: one from Paddle and one from Proton.
